.github/workflows/e2e_test.yml

```
@@ -0,0 +1,30 @@
+name: e2e_test
```

```suggestion
name: End-to-end Testing (Nightly)
```

(this one is supposed to the human friendly string)